deployment/helm/.gitignore

```diff
@@ -0,0 +1,37 @@
+# General files for the project
```

Comment:
Do we need another `.gitignore` if we already have a project level one? Can this be merged to the top level one?

Reply:
You are right. I also dislike having several `.gitignore` files. I just merged it with the root level `.gitignore`